.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,63 @@
+name: feature request
+description: Suggest an idea for this project
+title: "[Feature] "
+labels: ["feature"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+
+        Thank you for finding the time to propose new feature!
+
+        We really appreciate the community efforts to improve AmazeFileManager.
+
+  - type: checkboxes
+    attributes:
+      label: Search before asking
+      description: >
+        Please make sure to search in the [issues](https://github.com/TeamAmaze/AmazeFileManager/issues) first to see
+        whether the same feature was requested already.
+      options:
+        - label: >
+            I had searched in the [issues](https://github.com/TeamAmaze/AmazeFileManager/issues) and found no similar
+            feature requirement.
+          required: true
+
+  - type: textarea
+    attributes:
+      label: Description
+      description: A short description of your feature
+
+  - type: textarea
+    attributes:
+      label: Use case
+      description: What do you want to happen?
+      placeholder: >
+        Rather than telling us how you might implement this feature, try to take a
+        step back and describe what you are trying to achieve.
+
+  - type: textarea
+    attributes:
+      label: Related issues
+      description: Is there currently another issue associated with this?
+
+  - type: checkboxes
+    attributes:
+      label: Are you willing to submit a PR?
+      options:
+        - label: Yes I am willing to submit a PR!
+
+  - type: checkboxes
+    attributes:
+      label: Code of Conduct
+      description:
+        The Code of Conduct helps create a safe space for everyone. We require
+        that everyone agrees to it.
+      options:
+        - label: |
+            I agree to follow this project's [Code of Conduct](https://github.com/TeamAmaze/AmazeFileManager/blob/release/3.7/CODE_OF_CONDUCT.md)
+          required: true
```

Comment:
This should be auto accepted on submission, it shouldn't require user checking a box IMO, users don't check boxes when commenting, and still applies.